variables.tf

```diff
@@ -1,4 +1,118 @@
-variable "example" {
-  description = "Example variable"
-  default     = "hello world"
+variable "zone" {
+  type        = string
+  description = "The DNS zone name which will be added."
+}
+
+variable "zone_enabled" {
+  type        = bool
+  description = "Whether to create DNS zone otherwise use existing."
+  default     = true
+}
+
+variable "records" {
+  type        = list(any)
```

Comment:
consider using `set(object(...))` or `list(object(...))`

Reply:
I agree about using `any` instead of `object` prior to TF 0.15, but we should use `map` instead of `list` so we have sane, user-provided names.